main.go

```diff
@@ -52,13 +53,24 @@ func RunServer(logfile io.Writer) {
 	host := pflag.String("host", "0.0.0.0", "Host of the server")
 	tlsCertPath := pflag.String("tls-cert-path", "server.crt", "TLS Certificate path")
 	tlsKeyPath := pflag.String("tls-key-path", "server.key", "TLS Private key path")
+	statsdHost := pflag.String("statsd-host", "", "Metrics Host")
+	statsdPort := pflag.String("statsd-port", "", "Metrics port")
+	agentNamespace := pflag.String("agent-namespace", "agent.prod", "Agent Namespace")
```

Comment:
AgentNamespace is only used for Statsd. Would it make sense to limit its scope and rename the variable to `statsd-namespace`?